frontend/gist-frontend/src/Components/AboutGistComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { Clock, Heading, Box, Meter } from "grommet";
+
+export default class AboutGistComponent extends React.Component {
+  render() {
+    return (
+      <div>
```

Comment:
instead of `<div>...component stuff...</div>`, you can use `<>...component stuff... </>` for the outside parts of react components

Reply:
Oh you...